src/server/js/app.js

```diff
@@ -48,6 +49,9 @@ app.engine('hbs', hbs({
 app.set('views', path.join(nodeRoot, 'shared', 'views'));
 app.set('view engine', 'hbs');
 
+const serviceAccount = require('../../data/keys/pie-shop-app-firebase-adminsdk-cq5dk-ff2e1cd893.json');
```

Comment:
This should be a config variable or something similar. Or maybe just follow a convention like `firebase-config-{env}.json`.